.github/workflows/gradle-plugins.yml

```diff
@@ -35,5 +35,5 @@ jobs:
           SONATYPE_KEY: ${{ secrets.SONATYPE_KEY }}
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSWORD: ${{ secrets.GPG_PASSWORD }}
-        run: ../gradlew build snapshot
+        run: ../gradlew build publishToSonatype closeAndReleaseStagingRepository
```

Comment:
What's wrong with `snapshot`?

Reply:
This will publish to oss.sonatype right? No need to close repo for it (don't know if the API works).

Reply:
It is task provided by `nebula` plugin, which is not used in this module

Reply:
> This will publish to oss.sonatype right? No need to close repo for it (don't know if the API works).

I have no idea. I was unable to find any explanation what does which task do and how to just publish public artifact to `https://oss.sonatype.org/content/repositories/snapshots/`.

https://central.sonatype.org/publish/publish-guide/#releasing-to-central seems to say that after deployment the repo is private until it is closed.

Reply:
It's the nexus publish plugin that controls where things go. SNAPSHOT will go to oss.sonatype snapshots by default. No need to close it.

Otherwise Maven Central would reject them if they went there.

I'm not a huge fan of the separate settings.gradle and having to reproduce all our publishing logic, so couldn't approve the other PR. Perhaps this is motivation to avoid it? ;)

Reply:
Sure, as soon as we understand how to properly re-use all the existing code. I was unable to do so :)